source/dofs/dof_tools_constraints.cc

```diff
@@ -1083,6 +1085,77 @@ namespace DoFTools
         }
     }
 
+    namespace internal
+    {
+      /**
+       * Try to find a finite element inside FECollection of @p dof_handler
+       * which is dominating those in @fes .
```

Comment:
also, I think you want to say in the function name that you're looking for *face* domination.